core/color.cpp

```diff
@@ -335,36 +335,23 @@ Color Color::html(const String &p_color) {
 	} else if (color.length() == 6) {
 		alpha = false;
 	} else {
-		ERR_EXPLAIN("Invalid Color Code: " + p_color);
-		ERR_FAIL_V(Color());
+		ERR_FAIL_V_MSG(Color(), "Invalid Color Code: " + p_color + ".");
```

Comment:
If you can also fix capitalization for the errors in this class, that would be nice. "Invalid color code: "